src/main/java/io/fabric8/maven/docker/service/helper/StartContainerExecutor.java

```diff
@@ -41,39 +42,41 @@
 
     private StartContainerExecutor(){}
 
-    public String startContainers() throws IOException, ExecException {
+    public AbstractMap.SimpleEntry<String, Properties> startContainers() throws IOException, ExecException {
```

Comment:
I think we already include Apache Commons Lang, so we could use https://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/tuple/ImmutablePair.html here, which better declares the intent imo